multichain-testing/test/support.ts

```diff
@@ -151,12 +151,15 @@ export const commonSetup = async (
     contractBuilder: string,
     builderOpts?: Record<string, string | string[]>,
   ) => {
+    await null;
     const { vstorageClient } = tools;
-    const instances = Object.fromEntries(
-      await vstorageClient.queryData(`published.agoricNames.instance`),
-    );
-    if (contractName in instances) {
-      return t.log('Contract found. Skipping installation...');
+    if (contractBuilder.endsWith('/start-fast-usdc.core.js')) {
```

Comment:
this is a bit of a KLUDGE. Once I can tell that it's working, I expect to refactor it.